src/Bundle/Twig/Token/JavascriptTokenParser.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Hostnet\Bundle\WebpackBundle\Twig\Token;
+
+/**
+ * @author Harold Iedema <hiedema@hostnet.nl>
+ */
+class JavascriptTokenParser extends TokenParser
+{
+    /**
+     * Constant declaration for easy accessibility by the TwigParser component.
+     *
+     * @var string
+     */
+    const TAG = 'webpack_javascripts';
```

Comment:
What if you make this a generic `webpack` tag and allow a type?

```
{% webpack js '...' %}
    <script ...>
{% endwebpack %}


{% webpack css '...' %}
    <link ...>
{% endwebpack %}


{% webpack image '...' %}
    <img ...>
{% endwebpack %}
```

You could create "plugins" to extend automatically on this let plugins decide what to use. Maybe this is a bit over-engineered, but I see potential where something like `{% webpack image '...' size {100, 150} %}` is possible.